pennylane/operation.py

```diff
@@ -798,6 +850,53 @@ class Observable(Operator):
     # pylint: disable=abstract-method
     return_type = None
 
+    @classmethod
+    def _eigvals(cls, *params):
+        """Eigenvalues of the observable.
+
+        The order of the eigenvalues needs to match the order of
+        the computational basis vectors when the observable is
+        diagonalized using :attr:`diagonalizing_gates`.
+
+        This is a *class method* that must be defined for all
+        new diagonal operations, that returns the eigenvalues
+        of the operator in the computational basis.
+
+        This private method allows eigenvalues to be computed
+        directly without instantiating the operators first.
+
+        To return the eigenvalues of *instantiated* operators,
+        please use the :attr:`~.Operator.eigvals` property instead.
+
+        **Example:**
+
+        >>> qml.Z._eigvals()
```

Comment:
What is `qml.Z` in this case? Should it be `qml.RZ`, with parameters?

Reply:
```suggestion
        >>> qml.RZ._eigvals(0.5)
```

Reply:
Hoopsie, `PauliZ` it should be